### ISSUE TITLE
Reduce fleet frequencies in human space across the board?

### ISSUE THREAD

**Issue by Amazinite** [endless-sky/endless-sky#4039]
#4038 got me thinking; a critique I've seen a few times about the game is that systems are often *too* populated. [Here](https://docs.google.com/spreadsheets/d/1oZJLAlqxZwkJZtqya3egwe21U3yEkGt_VM8tCk8C6cI/edit?usp=sharing) is a spreadsheet showing the average fleet frequency for each human system (prior to any events) as well as the average number of seconds between fleet spawns. (Calculated by finding the inverse of the sum of the inverses of the fleet frequencies in a system, e.g. fleet 1 is 500, fleet 2 is 1000, 1/(1/500 + 1/1000) = 333.3, the average fleet frequency of the system).

As you can see, about 85% of all human systems see a fleet entering on average once every 10 seconds or less, while about 97% of all human systems see a fleet entering on average once every 20 seconds or less. Only five systems take longer for fleets to spawn than this, and other than Ultima Thule which has no fleet spawns, no human system takes longer than 48 seconds on average for a fleet to show up.

Arguments I can think of as to why we might want to reduce the fleet frequencies:
* There isn't much of a sense of exploration when every system you go to already has a steady stream of fleets entering and exiting. (The argument I've seen for this.)
* Big battles like during the campaign don't seem so special when hanging out near a pirate planet can result in a battle just as big rather often.
* A place like Coalition space has many times more inhabitants than human space in a much denser area, but the number of ships you see feels just like human space; reducing frequencies in human space could make alien region feel more special in this way.

And to end, here's a map of human space with systems color coded by how often fleets enter the system (prior to any changes to fleet spawns due to events). Sol is perfect blue and perfect black is for systems that take 10 seconds or longer on average for a fleet to spawn. (Ignore the boxed-in systems, that's stuff from plugins.) At least in terms of how the frequency of systems compare to one another, there are at least a couple of anomalies. (e.g. Algieba, an uninhabited system, being highly populated; Limen, an uninhabited system surrounded by uninhabited systems and near inhabited systems with very low frequencies, has a rather high fleet frequency; Sheratan neighbored on either side by rather high frequency systems, has a very low frequency; the system connecting Kraz to the rest of the galaxy has a lower frequency that Kraz itself and the other neighboring system, etc.)
![image](https://user-images.githubusercontent.com/17688683/49038754-d6207680-f18b-11e8-838e-254d8d18d2d3.png)

**Comment by Pointedstick** [comment]
I strongly agree. The generally high fleet spawn frequency also reduces the distinctiveness of the different regions of human space. Most of humanity and the intergalactic economy are supposed to be centered around Earth, the paradise planets, and the western edge of Syndicate space, but those areas don't seem that much busier in terms of fleet movements compared to a lot of the other parts of human space.

**Comment by solardawning** [comment]
Yes.

It's a very common new player point of confusion that 'enemy reinforcements keep coming'. The rapid rate of fleet spawning leads to neverending battle, and unless you're familiar with the mechanics, that misleads a lot of people into thinking that they are supposed to be fighting everything that appears while doing bounty missions or other content.

**Comment by Zitchas** [comment]
I think there needs to be a bit more scale, and a bit more variety in ship selection.

Most of human space (especially the core, paradise, and dirt belt) I think should stay just as busy as they are now, but there needs to be a bit more variety in the ships that spawn (especially the dirt belt should probably have more poor and/or tough ships.

It is mainly the uninhabited systems that seems over-populated. An empty system between major inhabited places? sure, I'd expect plenty of traffic. A dead end uninhabited system? Uh, why would *anyone* be out here?

**Comment by jafdy** [comment]
Keep in mind that we don't want to have the player stranded so we want almost all human systems to have at least a small amount of friendly traffic.

**Comment by nylki** [comment]
> Keep in mind that we don't want to have the player stranded so we want almost all human systems to have at least a small amount of friendly traffic.

As far as I remember this should not happen anymore since you have a small ramscoop capability by default since a couple of months ago, so even if you have to wait for a while for that jump to fill up, you can't really strand.

I support the idea to have empty ends being much more empty and only core systems and routes having higher traffic.

**Comment by Tadrix** [comment]
Maybe I'm too used to what we have now, but I'm against reducing frequencies. Tweaking them so they make more sense pattern-wise might be a better idea, but not reducing overall.

**Comment by Nescio0** [comment]
Another way to make systems appear less populated, is simply to increase the distances between planets. (That would also make thrusters more desirable.)

**Comment by Pointedstick** [comment]
Also increasing the warp-in distances from the system center could help a lot. This would also make fleet battles less likely to instantly devolve into a huge brawl in the center--or at last, increase the amount of time that passes before this happens.

**Comment by jafdy** [comment]
nylki, I'm well aware of that ramscoop, it was added when the korath automata were to prevent the player getting stranded in that large empty space. It also takes a really long time to get a jump with that ramscoop. 
The main reason I said that we don't want empty human systems is so that a new player isn't going to be forced to wait for that ramscoop to do it's work, especially as new players are more likely to get stranded like that.
